code/modules/projectiles/guns/rifles.dm

```diff
@@ -1274,9 +1274,6 @@
 	starting_attachment_types = list(/obj/item/attachable/stock/carbine/wood/drill)
 	map_specific_decoration = FALSE
 
-/obj/item/weapon/gun/rifle/l42a/abr40/set_gun_config_values()
-	..()
-	fire_delay = FIRE_DELAY_TIER_7
```

Comment:
This can stay, it's slower than base L42A.

Reply:
Yeah the base ABR-40 is intended to be a surv weapon, though it is unused currently as I am seeking another sprite for it.

Reply:
Fixed